ui/src/components/TableRow.js

```diff
@@ -0,0 +1,98 @@
+import React from 'react';
+import styled from 'styled-components';
+import { Tooltip } from '@scality/core-ui';
+import { fontWeight, padding } from '@scality/core-ui/dist/style/theme';
+import { intl } from '../translations/IntlGlobalProvider';
+
+const TableRowStyle = styled.div`
+  &:hover,
+  &:focus {
+    background-color: ${(props) => props.theme.brand.backgroundBluer};
+    outline: none;
+    cursor: pointer;
+  }
+
+  &:last-child {
+    border: none;
+  }
+
+  background-color: ${(props) =>
+    props.volumeName === props.row.values.name
+      ? props.theme.brand.backgroundBluer
+      : props.theme.brand.primary};
+`;
+
+const TooltipContent = styled.div`
+  color: ${(props) => props.theme.brand.textSecondary};
+  font-weight: ${fontWeight.bold};
+  min-width: 60px;
+`;
+
+const UnknownIcon = styled.i`
+  color: ${(props) => props.theme.brand.textSecondary};
+  // Increase the height so that the users don't need to hover precisely on the hyphen.
+  height: 30px;
+  padding-top: ${padding.base};
+`;
+
+const TableRow = (props) => {
+  const { row, style, rowClicked, volumeName, theme } = props;
```

Comment:
I would rename `rowClicked` props to `onClick` to match standard event names

Reply:
Instead of passing volumeName I would introduce an isSelected prop that reflect in a clearer way if the row is selected or not

Reply:
In order to avoid props drilling of the theme you can rely on styled component which automatically map the theme as part of the props or use `useTheme` hook that retrieve current theme from styled-component internal context